library/src/containers/AsyncApi/AsyncApi.tsx

```diff
@@ -23,6 +24,7 @@ export interface AsyncApiProps {
 interface AsyncApiState {
   validatedSchema: AsyncApi;
   validated: boolean;
+  error: Error | undefined;
```

Comment:
error?: Error would be better, unless this is not working.

Reply:
without undefined it is not working